deployments/common/image/common-env/jupyter.pip

```diff
@@ -25,3 +25,5 @@ ipyvolume
 ipyvuetify
 ipywebrtc
 ipykernel
+jupyter-resource-usage
+git+https://github.com/jaytmiller/nersc-refresh-announcements@octarine-updates
```

Comment:
Just checking that you want to include from your own fork

Reply:
Actually,  no,  nice catch!  But I fix it in the next PR.